AUTHORING_MANIFESTS.md

```diff
@@ -77,3 +86,135 @@ It is important to test your manifest before submission to ensure it meets the r
 
 After enabling the setting for local manifests (`winget settings --enable LocalManifestFiles`), manifests can be tested locally with `winget install --manifest <path>`.
 If your system supports Windows Sandbox, you can also use the [SandboxTest.ps1 Script](https://github.com/microsoft/winget-pkgs/blob/master/doc/tools/SandboxTest.md) to test the manifest in the Windows Sandbox. This is the preferred method, as it ensures the package doesn't require any dependencies to install.
+
+## Version Matching & Correllation
+
+The goal is to accurately determine the currently installed version, and all available versions of this application, so upgrades can be correctly suggested when available, and not suggested when not available. Most installers write accurate version data to the Windows Registry. But some do not, and `Portable` type applications generally cannot as they usually don't have installers. The data available to meet this goal consists of the files on disk, data in Windows Registry, and manifests in the winget-pkgs repo.
+
+### Metadata 
+
+- DisplayName
+- DisplayVersion
+  - Displayed in the Control Panel's Add & Remove Programs page (ARP) and/or Apps and features Settings page (ANF). 
+  - Should not be used if it's always equal to PackageVersion.
+  - Must be unique per package. if multiple versions are sharing the same DisplayVersion, we may only keep the 'latest' one in the repo.
+- Package Familyname (MSIX)
```

Comment:
This isn’t a valid AppsAndFeaturesEntry field and can only be used outside of the AppsAndFeaturesEntries